baseline/baseline

```diff
@@ -0,0 +1 @@
+date=01-16-20,time=17:04:57,repo=baseline-chaos-operator,branch=master,commit=3280b9d6
```

Comment:
Can we do something like that -
-   We will maintain this file in such a way that it will only contain the latest records of all repositories 
<table> <tr> 
<th> Repo name </th>
<th> date </th>
<th> time </th>
<th> branch  </th>
<th> commit sha</th>
</tr> </table>

-  For keep track ( if some failure occur) we can maintain a separate file which contains the history of all repository of the entire day

Reply:
Anyways we have to keep all records we can do this with one file only and we will find out the latest record from this file can using some script.

Reply:
I don't think this file requires now anyway ansible will create it if not present

Reply:
yes @chandankumar4

Reply:
okay